src/properties/propertiesService.ts

```diff
@@ -23,132 +18,71 @@ export interface PropertiesService {
   ): PropertySchemas | undefined;
 }
 
-interface PropertySchemas {
+/**
+ * Schemas describing the properties view of an editor element.
+ */
+export interface PropertySchemas {
   schema: JsonSchema;
   uiSchema?: UISchemaElement;
 }
 
-const initialSchemas = (): PropertySchemas => ({
-  schema: {
-    type: 'object',
-    properties: {},
-  },
-  uiSchema: {
-    type: 'VerticalLayout',
-    elements: [],
-  } as Layout,
-});
-
-const createControl = (controlScope: string): ControlElement => ({
-  type: 'Control',
-  scope: controlScope,
-});
-
-interface PropertiesSchemasDecorators {
+/**
+ * Decorator for the PropertySchemas of an EditorUISchemaElement.
+ */
+export interface PropertiesSchemasDecorator {
```

Comment:
do we expect this interface to provide more than this one function? if not maybe convert to a function interface?